### PR TITLE
bpo-41026: Add PathLike object support to Mailbox constructor 

### PR DESCRIPTION
**Pull request by srinivasreddy**
https://bugs.python.org/issue41026

### CODE REVIEW

**Comment by loz-hurst** [comment]
Hi,

I'm still trying to get my CLA sorted out (I signed it last week but I've been told they cannot find my BPO username despite it being correct on the CLA and BPO).  Until this gets sorted out, I don't think you can merge this PR since you have copied and pasted my work (https://github.com/python/cpython/pull/20976, 793ad9a "Copy blurb and docs from @loz-hurst" - copying my work and removing me from the ACKS is both rude and plagiarism (since the commit message will get squashed when merged, losing all attribution of my contribution).

In short: **This PR contains work that has not been contributed under the CLA yet.**

Laurence

**Comment by srinivasreddy** [comment]
I closed this PR. Don’t worry.

**Comment by loz-hurst** [comment]
That's a shame, yours was more comprehensive than mine.  I was only objecting to you copy and pasting parts of my work and removing the attribution of the work.

You do need to be careful about copying work that's not been submitted with a valid CLA and submitting it under your name which does, it could get Python into legal hot-water if it got merged.
